turbopack/crates/turbopack-core/src/module_graph/chunk_group_info.rs

```diff
@@ -327,10 +322,7 @@ impl Ord for NodeWithPriority {
         // Smaller group length has higher priority
```

Comment:
why? naively i would expect that larger chunk groups have priority since we would want them to reach a fixed point sooner?